### DISCUSSION TITLE
Can't download Youtube movie. VDH greyed out.

### DISCUSSION THREAD

**Discussion by rw330i** [aclap-dev/video-downloadhelper#944] [Help and Q&A] [answered]
### Your question

I've never tried to download a Youtube movie before so I don't know if VDH is supposed to be able to download them. For example https://www.youtube.com/watch?v=hr7WWqynnxw. 
Same result in both Firefox & Chrome.
Thank you

Just realized you may not be able to see the link unless you have a premium Youtube subscription.

### Addon & CoApp version and details

version: 9.0.2.12
channel: stable
build date: 2024-05-30
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: accepted
key (only 16 first characters): 4152042dea214cef
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:127.0) Gecko/20100101 Firefox/127.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\Richard",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by jcfrog** [comment]
Hi,
Not available in my country but I guess there might be DRMs.

**Reply by GrampaWildWilly** [reply]
![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/a09a1560-c5b1-4f75-b9f8-a010bd737f02)

**Comment by rw330i** [comment, marked as answer]
Duh! I didn't consider that they might be DRM protected. Thank you for your help GrampaWildWilly!
